### PR TITLE
form for name params created and revised index file

### PR DESCRIPTION
**Pull request by rellbows**
@samuraiseoul

### CODE REVIEW

File: NameGame.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Name Game</title>
+	</head>
+	
+	<body>
+		<h1>Please enter you name below...</h1>
+			<form action="index.php" method="get">
+					First Name: <input type="text" name="name" /><br />
```
Comment (samuraiseoul) [inline NameGame.php]: Consider changing the name from just name to firstname to be more clear.

File: index.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	<body>
 		<h1>
 			<?php 
-				if(isset($_GET['name']) && isset($_GET['lastname'])) { 
+				if(isset($_GET['name']) && !empty($_GET['name']) && isset($_GET['lastname']) && !empty($_GET['lastname'])) {
```
Comment (samuraiseoul) [inline index.php]: consider wrapping the checks for isset() and !empty in a function, maybe call it isPresent()

**Comment by rellbows** [comment]
@samuraiseoul

File: index.php [hunk 3]
```diff
@@ -4,9 +4,12 @@
 	</head>
 	<body>
 		<h1>
-			<?php 
-				if(isset($_GET['name']) && isset($_GET['lastname'])) { 
-					echo "Hello " . $_GET['name'] . " " . $_GET['lastname'] . "!";
+			<?php
+				function isPresent($var) {
+					return isset($var) && !empty($var);
+					}
```
Comment (samuraiseoul) [inline index.php:10]: spacing here, the closing } should be inline with the word "function"

File: NameGame.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Name Game</title>
+	</head>
+	
+	<body>
+		<h1>Please enter you name below...</h1>
+			<form action="index.php" method="get">
```
Comment (samuraiseoul) [inline NameGame.php:9]: spacing here, the `<form>` and sub tags should be on the same spacing level as the `<h1>` because they are the same element depth.

**Comment by samuraiseoul** [comment]
Looks like you didn't fix the tabbing issue on index.php still. Should not have merged.

**Comment by rellbows** [comment]
Yeah, you're right. I hadn't taken a look at this in a while, and thought it was good at first glance. I'll go in and update those per your markups. 1 quick question about the spacing in the form and its sub tags. Should the sub tags of the `form` be indented?

**Pull request merged**